### PR TITLE
Helm flag Mapping Function

### PR DESCRIPTION
**Pull request by mphammer**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1257#" title="Author self-approved">mphammer</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by mphammer** [comment]
Despite the test cases that I made, I don't think we should merge this into 2020.2.0 since it's being release in 4 days.

File: pkg/polaris-reporting/helpers.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ *  under the License.
+ */
+
+package polarisreporting
+
+// SetHelmValueInMap adds the finalValue into the valueMap at the location specified
+// by the keyList
+// keyList - an ordered list of keys that lead to the location in the valueMap to place the finalValue
+func SetHelmValueInMap(valueMap map[string]interface{}, keyList []string, finalValue interface{}) {
+	currMap := valueMap
```
Comment (msenmurugan) [inline pkg/polaris-reporting/helpers.go]: why we need this?

File: pkg/polaris-reporting/helpers.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	for i := range keyList {
+		currKey := keyList[i]
```
Comment (msenmurugan) [inline pkg/polaris-reporting/helpers.go]: can you pls replace with for i, currKey := range KeyList {

File: pkg/polaris-reporting/helpers.go (hunk 1 above)
Comment (mphammer) [reply pkg/polaris-reporting/helpers.go]: I was doing to for "code readabilty" and clarity. But I renamed some variables to make it clear

File: pkg/polaris-reporting/helpers.go (hunk 2 above)
Comment (mphammer) [reply pkg/polaris-reporting/helpers.go]: Done!

**Pull request merged**